Networking/Networking/Model/Order.swift

```diff
@@ -239,6 +245,9 @@ public struct Order: Decodable, Sendable, GeneratedCopiable, GeneratedFakeable {
             return OrderAttributionInfo(metaData: allOrderMetaData)
         }()
 
+        // Shipping labels
+        let shippingLabels = try container.decodeIfPresent([ShippingLabel].self, forKey: .shippingLabels) ?? []
```

Comment:
👋 @pmusolino  Do we receive this `shipping_labels` key in orders response? Based on my testing in a Woo store with shipping labels, I don't see it being sent in `wc/v3/orders`. 

If we do receive the `shipping_labels`, should we update the mock JSON and add unit tests to ensure that `shipping_labels` are mapped correctly?

Reply:
Good question, @selanthiraiyan. In `ShippingLabelStore`, the method `upsertShippingLabels` connects the fetched shipping labels to the specific order, so the order doesn't return specific shipping labels, making it unnecessary to decode the Shipping Labels in the Order model. However, declaring the property (without decoding) in the networking model seems necessary to access `shippingLabels`, unless you have another solution in mind. 

Yes, we could access the shipping labels from storage, but that's what we're trying to avoid. Do you think I can leave the `shippingLabels` property in `Order.swift` but remove the decoding? This way, by default, it would be an empty array.

Reply:
Thanks, @pmusolino! I don't have a solution in mind. But I think we should remove the decoding, as it will be misleading. 

I'm tagging @itsmeichigo to check whether she has any other solutions.

Reply:
Thanks for the ping Sharma!

I think if the key for shipping labels is non-existent in the order response, keeping the property an empty array should be the way to go. There's no need to waste more resources to look for such a key to decode.

However, I'm concerned about this part:

> In ShippingLabelStore, the method upsertShippingLabels connects the fetched shipping labels to the specific order

In the order details screen, after syncing shipping labels, we'd need to ensure that the `order` in the view model is updated with the fetched label(s), just so they can be displayed on the UI.

Reply:
Thank you both! I updated the decoding here 85c0115c456026ec9e6865542d2513a3887c5d36 the comment should be clear enough.

> In the order details screen, after syncing shipping labels, we'd need to ensure that the order in the view model is updated with the fetched label(s), just so they can be displayed on the UI.

Yes, this already happens from my testing. If you create or update a shipping label, or open an order detail with an existing shipping label that needs to be fetched remotely, you will see the order detail UI correctly updated.